Projects/My_website/myWebsite/blog/serializers.py

```diff
@@ -1,9 +1,22 @@
 from rest_framework import serializers
-from .models import Post
+from .models import Post,Tag,User
 
 
-class blogSerializer(serializers.ModelSerializer):
+class TagSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Tag
+        fields = ['id','caption']
+
+class AuthorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = ['first_name',"last_name"]
```

Comment:
make consistent in commas '' or ""
Use code formatting tool to format python code.